gradlew

```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
```

Comment:
I know you said that this is for @sb-ebukaanene. But can this be generated when needed instead of in the repository.

Reply:
It's definitely possible but that would be somewhat of a circular thing. Gradle wrapper is used to tie app building to a specific gradle version and support building without installing gradle first. 
 Without this everyone would need gradle installing and we would need to rely on their version to generate gradlew using the version we want.